pydantic/main.py

```diff
@@ -17,6 +17,9 @@ class BaseConfig:
     allow_extra = False
     fields = None
 
+    # tagged on here to make avoid the need for extra imports, e.g. you can just use `BaseModel.Config.Required`
+    Required = Required
```

Comment:
Don't think that works here. `Required` is already added to `__init__`, this is to avoid having to import it at all.